process/factory/shard/interceptorsContainerFactory_test.go

```diff
@@ -385,6 +385,31 @@ func TestNewInterceptorsContainerFactory_NilAddrConverterShouldErr(t *testing.T)
 func TestNewInterceptorsContainerFactory_NilTxFeeHandlerShouldErr(t *testing.T) {
 	t.Parallel()
 
+	icf, err := shard.NewInterceptorsContainerFactory(
```

Comment:
Test name and what the test does do not match. This should be NilBlackListHandlerShouldErr, and the next one NilTxFeeHandlerShouldErr.

Reply:
refactored